model/validation.go

```diff
@@ -806,6 +806,28 @@ func ValidateProxyAddress(hostAddr string) error {
 	return nil
 }
 
+// ValidateIPv4AddressAndPort checks that a network address is well-formed.
+// This works just like ValidateProxyAddress, but without support for FQDNs.
```

Comment:
This was needed because the statsd udp address (as currently processed by Envoy) has to be <IP>:<Port>. So, the thinking was to have some validation of something that can only be an IP Addr.